src/Compilers/CSharp/Portable/Symbols/Source/SourceAssemblySymbol.cs

```diff
@@ -661,13 +661,9 @@ private static void ReportDiagnosticsForNullableAttribute(CSharpCompilation comp
 
         internal static DiagnosticInfo GetUseSiteDiagnosticForNullableAttribute(CSharpCompilation compilation)
         {
-            if ((object)compilation.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_NullableAttribute__ctor) == null ||
-                (object)compilation.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_NullableAttribute__ctorTransformFlags) == null)
-            {
-                return new CSDiagnosticInfo(ErrorCode.ERR_NullableAttributeMissing, AttributeDescription.NullableAttribute.FullName);
-            }
-
-            return compilation.GetSpecialType(SpecialType.System_Boolean).GetUseSiteDiagnostic();
```

Comment:
>compilation.GetSpecialType(SpecialType.System_Boolean).GetUseSiteDiagnostic(); [](start = 19, length = 78)

This check still looks relevant. Also, consider keeping the other check commented out so that it would be available for when time comes to address the PROTOTYPE comment.

Reply:
Added back previous checks, some commented out.